tests/test_ec2/test_network_acls.py

```diff
@@ -7,41 +7,37 @@
 
 @mock_ec2
 def test_default_network_acl_created_with_vpc():
-
     conn = boto.connect_vpc('the_key', 'the secret')
     vpc = conn.create_vpc("10.0.0.0/16")
-
     all_network_acls = conn.get_all_network_acls()
-    all_network_acls.should.have.length_of(1)
+    all_network_acls.should.have.length_of(2)
```

Comment:
Sorry for not catching up to this sooner!

So this is because every time a new VPC is created, the default ACL for that VPC is created as well:

https://github.com/spulec/moto/blob/master/moto/ec2/models.py#L1832-L1833

With the change in this PR, the default VPC (and the corresponding default network ACL) already exists when the mock starts. But in this test case, a new VPC and (its default ACL) is explicitly created. So in total, there are two network ACLs found with `get_all_network_acls()` since by default the method list all the ACLs found in the region.

In general, I think I had to increment the count of ACLs by one for tests of this kind due to the pre-existing default VPC. So this could be confusing depending on the assumption. The original intention might be clearer if the tests get network ACLs associated only with the VPC newly created within the test?